unittest/python/test_contacts_2D.py

```diff
@@ -0,0 +1,92 @@
+import numpy.core.multiarray
+import os
+import sys
+
+import numpy as np
+
+import pinocchio
+from pinocchio.utils import rand,zero
+import crocoddyl
+import matplotlib.pylab as plt
+from numpy.linalg import norm, solve
+
+sys.path.append("/local/src/crocoddyl/unittest/")
```

Comment:
you should remove these custom paths. Moreover, I don't think you need to import sys here.
Have a look at `test_contacts.py`. It is legacy code, but it might show you things you need. For example, you don't need to import matplotlib in a unittest